### PR TITLE
Functionality for use with GOV.UK Notify API

### PR DESCRIPTION
**Pull request by davelovemartin**

### CODE REVIEW

**Comment by davelovemartin** [comment]
2 Functions: `notifyByEmail` and `notifyBySms` which take a `userData` parameter.  Dispatches `sendNotification` which persists the `userData` to the state and calls the relevant `NotifyClient` method.  

Template IDs are currently hardcoded in these functions. [Create new Templates](https://www.notifications.service.gov.uk/services/63998e44-a39f-4f58-b2a7-bb8f7a4269e1/templates).

[Create a new API key](https://www.notifications.service.gov.uk/services/63998e44-a39f-4f58-b2a7-bb8f7a4269e1/api) and store in config.development.yaml as `NOTIFY_API_KEY`

File: config.development.yaml [hunk 1]
```diff
@@ -2,3 +2,5 @@ api: http://127.0.0.1:3000
 buildConfig: development
 
 users: !!import/single ./config.users.yaml
+
+NOTIFY_API_KEY: initial_test_2-63998e44-a39f-4f58-b2a7-bb8f7a4269e1-2075e450-b620-4b8f-ac60-32d0427e6a44
```
Comment (Koded) [inline config.development.yaml]: Hi @justan0therdave - we can't have api keys in the repo. I will talk you through this when you get back.

**Review by Koded — APPROVED** [review]
Hi @justan0therdave 

All looking good. I've removed the `config.development.yaml` file. It was fine in there up until now, but you added a gov.uk notify key, so i've removed it from the repo. I've also deleted the key from gov Notify.

We can get the key on the server without putting it in the repo. I'll get that sorted out. I can run through this with you when you get back.

**Comment by Koded** [comment]
Just a further note - I know you opened up a PR to remove the config files. This was a good spot. They don't have any sensitive information in them currently, so they are ok for now, but will put the configuration on the server.

**Comment by enginehouse** [comment]
@Koded - can you run through this with me too? Not sure where you want this.

**Pull request merged**